platform/windows/os_windows.cpp

```diff
@@ -2586,7 +2586,12 @@ Error OS_Windows::execute(const String &p_path, const List<String> &p_arguments,
 	modstr.resize(cmdline.size());
 	for (int i = 0; i < cmdline.size(); i++)
 		modstr.write[i] = cmdline[i];
-	int ret = CreateProcessW(NULL, modstr.ptrw(), NULL, NULL, 0, NORMAL_PRIORITY_CLASS, NULL, NULL, si_w, &pi.pi);
+	int ret;
+	if (p_hide_cmd) {
+		ret = CreateProcessW(NULL, modstr.ptrw(), NULL, NULL, 0, NORMAL_PRIORITY_CLASS, NULL, NULL, si_w, &pi.pi);
```

Comment:
You could reduce duplication somewhat with:
```cpp
int flags = p_hide_cmd ? NORMAL_PRIORITY_CLASS & CREATE_NO_WINDOW : NORMAL_PRIORITY_CLASS
int ret = CreateProcessW(NULL, modstr.ptrw(), NULL, NULL, 0, flags, NULL, NULL, si_w, &pi.pi);
```

Reply:
Cool, I was wondering if that is possible!